### DISCUSSION TITLE
How to hide this

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3710] [General]
using something like this to make it less intrusive
 { filter = { event = "msg_show", find = "^%[Neo%-tree INFO%].*" }, view = "mini" },

but doesnt seem to be working

![image](https://github.com/LazyVim/LazyVim/assets/8222059/b0eaf9ec-0463-4cf0-a678-676d1554e024)

**Comment by folke** [comment]
That's not a `msg_show`, but a notification (event = notify)

**Reply by rubiin** [reply]
Not all heroes wear capes  , that did the trick

![](https://media1.tenor.com/m/PcwrxT_o6CgAAAAC/salute.gif)
